pkg/storage/doc.go

```diff
@@ -0,0 +1,11 @@
+// Package storage implements the storage layer of the application.
+// It contains the database operations for the application.
+// It interacts with the model layer to perform CRUD operations on the database.
+// It provides an abstraction over the database operations to the service layer.
+//
+// The storage layer should be used by the service layer to perform database operations.
```

Comment:
why just service layer? what if i want to use it somewhere else? why do you have such limitations?